invenio_communities/notifications/builders.py

```diff
@@ -216,3 +217,73 @@ class SubCommunityDecline(SubCommunityBuilderBase):
     """Notification builder for subcommunity request decline."""
 
     type = f"{SubCommunityBuilderBase.type}.decline"
+
+
+class SubComInvitationBuilderBase(SubCommunityBuilderBase):
+    """Base notification builder for subcommunity invitation requests."""
+
+    type = "subcommunity-invitation-request"
+
+    recipients = [
+        CommunityMembersRecipient("request.created_by", roles=["owner", "manager"]),
+        CommunityMembersRecipient("request.receiver", roles=["owner", "manager"]),
+    ]
```

Comment:
```suggestion
    recipients = [
        CommunityMembersRecipient("request.receiver", roles=["owner", "manager"]),
    ]
```

minor: shouldn't it be that only the receiver (i.e. the invited subcommunity) receives a notification about the request?

Reply:
kind of, but actually this is useful for fixing the email thing. I was trying to do it via recipient filters and wasn't having much luck